### PR TITLE
Add note about Ubuntu fuse dependency for Etcher

### LINKED ISSUE #24351: Complete instructions of making an HAOS image via Ubuntu / BalenaEtcher
### Feedback

I was unable to perform the tutorial for installing HAOS on a x86 system, by using method 2 (using a live operating system)

This guide points to an ubuntu USB bootable stick, in my case version 22.04
Downloading and extracting BalenaEtcher was easy
Exectuting however was not. Double clicking resulted in nothing

After a LOT of troubleshooting and help, i found that on the Ubuntu USB loader, the (right version of?) fuse was missing. I resolved it by performing this method: 

1. download and extract balenaetcher as described in the guide
2. open terminal (ctrl-alt-T)
3. Install fuse by executing the following commands:
sudo add-apt-repository universe
sudo apt install libfuse2
4. change to the directory to where BalenaEchter is extracted using "CD" commands to change directory and "LS" to list the files and folders (standard is Downloads folders)
5. Run the appimage as root by executing the following command:
sudo ./balenaEtcher-1.7.9-x64.AppImage
(or the litteral filename of the extracted appimage if the version is different)
6. Flash the boot disk of the system with HAOS as per the existing tutorial

As this was performed (and repeated) with a 'standard' ubuntu live OS 22.04 and the standard balenaetcher, I think everybody will encounter this issue the same (safe assumption?) and it may be worth to update the guide with this info. It cost me 5 hours to resolve (because it is hard to google and understand which steps of all possible solutions asre most feasible in linux if you have little to no experiece) 

### URL

https://www.home-assistant.io/installation/generic-x86-64

### Version

Ubuntu 22.04, Balena Etcher 1.7.9-x64

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ikifar2012**
## Proposed change

Added note about Ubuntu fuse dependency for Etcher to make it clearer for beginners 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #24351

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/installation/operating_system.md [hunk 1]
```diff
@@ -91,6 +91,18 @@ To use this method, follow the steps described in the procedure below: [Write th
 
 - To use this method, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
 
+<div class='note'>
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]: Let's not add a big note block for this. I think it is more fitting to put this in a details block instead.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/installation/operating_system.md [hunk 2]
```diff
@@ -91,6 +91,21 @@ To use this method, follow the steps described in the procedure below: [Write th
 
 - To use this method, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, follow the steps described in the procedure below: [Write the image to your boot media](#write-the-image-to-your-boot-media).
 
+<details>
```
Comment (frenck) [inline source/_includes/installation/operating_system.md]: Aa right, that is also a details thingy 😄 

I meant `{% details "title" %} ...content... {% enddetails %}`

Comment (ikifar2012) [reply source/_includes/installation/operating_system.md]: Awesome, did not know about this, it looks much better now

**Review by frenck — APPROVED** [review]
Thanks, @ikifar2012 👍

**Pull request merged**